scripts/pagerduty_notify.rb

```diff
@@ -82,6 +86,16 @@ def check_arguments(arglist)
 ap output_data if(ENV['debug'])
 #JSON.dump(output_data)
 
+disable_flag=false
+
+if ENV['disable_communications']
+  disable_flag=$store.substitute_string(ENV['disable_communications'])
+  if disable_flag.downcase=='yes' or disable_flag.downcase=='true' or disable_flag.downcase=='disable'
+    puts "-WARNING: Output to pagerduty is disabled in route config.  Change the <disable_communications> flag to allow communication."
```

Comment:
yeah - maybe - was thinking it would be worth highlighting in the log though, in case it's accidentally been set and someone is looking through trying to work out why pagerduty appears to be broken....